sdk/typescript/src/rpc/client.ts

```diff
@@ -53,7 +53,7 @@ export class JsonRpcClient {
               }
               if (key === 'balance') return value.toString();
               try {
-                if (value.isLosslessNumber) return value.valueOf();
+                if (value && value.isLosslessNumber) return value.valueOf(); // timestamp_ms: null
```

Comment:
If `value` is null, it should be returned on line 52. This error may have been fixed by changing `timestamp_ms: number | null;` to `timestamp_ms: number` https://github.com/MystenLabs/sui/blob/e7778f4af425a909c4f453eb2c5104ccead64f54/sdk/typescript/src/types/transactions.ts#L94 in https://github.com/MystenLabs/sui/commit/72b7133e6d73cef9c5c0be8102a8432c46da7f7d. Let me know if you are still able to repro!

Reply:
Oh, I understand. I'm testing, and check npm package version only. It's a mistake.
And I think It can be reduce returns.
if `if (value == null) { return value; }` is removed, check will be done in `try catch` only.

Reply:
if `value` is null, the parser should return `null` instead of `key`. We will be releasing a new version of SDK to npm registry soon today